tests/acceptance/pull-requests-test.js

```diff
@@ -139,9 +139,18 @@ module('Acceptance | pull-requests', function(hooks) {
     await settled();
 
     assert.dom('[data-test-clipboard-text]')
-      .hasText(
-        '<a href="https://github.com/bendemboski" rel="noopener noreferrer" target="_blank">Ben Demboski (@bendemboski)</a>, <a href="https://github.com/chadhietala" rel="noopener noreferrer" target="_blank">Chad Hietala (@chadhietala)</a>, <a href="https://github.com/cloke" rel="noopener noreferrer" target="_blank">Cory Loken (@cloke)</a>, and <a href="https://github.com/rwjblue" rel="noopener noreferrer" target="_blank">Robert Jackson (@rwjblue)</a>',
+      .containsText(
+        '<a href="https://github.com/bendemboski" rel="noopener noreferrer" target="_blank">Ben Demboski (@bendemboski)</a>');
+    assert.dom('[data-test-clipboard-text]')
+      .containsText(
+        '<a href="https://github.com/chadhietala" rel="noopener noreferrer" target="_blank">Chad Hietala (@chadhietala)</a>');
+    assert.dom('[data-test-clipboard-text]')
+      .containsText(
+        '<a href="https://github.com/cloke" rel="noopener noreferrer" target="_blank">Cory Loken (@cloke)</a>');  
+    assert.dom('[data-test-clipboard-text]')
+      .containsText(
+        '<a href="https://github.com/rwjblue" rel="noopener noreferrer" target="_blank">Robert Jackson (@rwjblue)</a>',
```

Comment:
To improve readability, let's simplify these lines by chaining the assertions. I think we can skip checking the `href` value because there's a unit test for this.

```suggestion
      .includesText('Ben Demboski (@bendemboski)', 'We see Ben Demboski.')
      .includesText('Chad Hietala (@chadhietala)', 'We see Chad Hietala.')
      .includesText('Cory Loken (@cloke)', 'We see Cory Loken')
      .includesText('Robert Jackson (@rwjblue)', 'We see Robert Jackson.');
```